wheels/MarkupSafe/meta.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+name: MarkupSafe
+version: 1.0
+build: 1
```

Comment:
@natefoo Should I use 2 here? I thought builds start at 0, but in the Travis logs this seems to be ignored.

Reply:
Or maybe it's fine to overwrite the old wheels?

Reply:
The build number isn't used for anything currently, other than to cause the file to be considered as modified by git, so that Travis will cause the wheel to be built when the only changes you've made are external (e.g. to Starforge). The wheels built will still be the same version as before.

Reply:
Ah, OK, I've dropped it.